### PR TITLE
[ID] Add The Team and group articles

### PR DESCRIPTION
**Pull request by hikikuman**
Add Indonesian translation for Nomination Assessment Team, Developers, and Project Loved Team. Update outdated Indonesian translation for Global Moderation Team, Beatmap Nominators, Support Team, and osu! Alumni. Add WalterToro to The Team.

### CODE REVIEW

**Comment by Nivalyx** [comment]
i reeeeeeeally wouldn't recommend translating the likes of `Global Moderation Team` into `Tim Global Moderasi` or `Nomination Assessment Team` into `Tim Penilai Nominasi` tho, it's much better to leave them as literally no one is calling them by their translated name really

also if you check on their respective usergroups (http://osu.ppy.sh/groups/4 and http://osu.ppy.sh/groups/7) you can see that they are still being touted as `Global Moderation Team` and `Nomination Assessment Team` regardless of what language you're currently using on the website for a good reason

**Comment by hikikuman** [comment]
> i reeeeeeeally wouldn't recommend translating the likes of `Global Moderation Team` into `Tim Global Moderasi` or `Nomination Assessment Team` into `Tim Penilai Nominasi` tho, it's much better to leave them as literally no one is calling them by their translated name really
> 
> also if you check on their respective usergroups (http://osu.ppy.sh/groups/4 and http://osu.ppy.sh/groups/7) you can see that they are still being touted as `Global Moderation Team` and `Nomination Assessment Team` regardless of what language you're currently using on the website for a good reason :(

Agree about that. As referenced from the outdated translation, I change all of that name, but I don't think it fits if the name is translated. Gonna check and change it back again now. Thank you, Niva! c:

File: wiki/People/The_Team/Developers/id.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Developer osu! dapat dikenali dari warna penggunanya yaitu pink dan gelar "osu! dev" atau "osu! team".
+
+## Anggota Tim
+
+Halaman daftar tim [Developer](https://osu.ppy.sh/groups/11).
+
+Upaya pengembangan osu! dipimpin oleh ![][flag_AU] **[peppy](https://osu.ppy.sh/users/2)**, sang pencipta osu!.
+
+### Pengembang Klien Game
```
Comment (troke12) [inline wiki/People/The_Team/Developers/id.md:22]:
```suggestion
### Developer
```

File: wiki/People/The_Team/Developers/id.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- ![][flag_AU] [nekodex](https://osu.ppy.sh/users/102)
+- ![][flag_JP] [notbakaneko](https://osu.ppy.sh/users/10751776)
+- ![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+
+### Desainer
+
+- ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765)
+- ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202)
+
+### Manajer Komunitas
```
Comment (troke12) [inline wiki/People/The_Team/Developers/id.md:43]:
```suggestion
### Community Manager
```

File: wiki/People/The_Team/Developers/id.md (hunk 1 above)
Comment (hikikuman) [reply wiki/People/The_Team/Developers/id.md:22]: oke

File: wiki/People/The_Team/Developers/id.md (hunk 2 above)
Comment (hikikuman) [reply wiki/People/The_Team/Developers/id.md:43]: yang lainnya perlu diubah gak? seperti yang desainer, akun bot, pengembang situs web dan server, sama yang anggota tim.

Comment (troke12) [reply wiki/People/The_Team/Developers/id.md:43]: kalau itu gak usah, udh pas sih

Comment (hikikuman) [reply wiki/People/The_Team/Developers/id.md:43]: oke2

File: wiki/People/The_Team/Project_Loved_Team/id.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+- ![][flag_US] **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**
+- ![][flag_US] [-mint-](https://osu.ppy.sh/users/8976576)
+- ![][flag_SG] [Abraxos](https://osu.ppy.sh/users/5025064)
+- ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573)
+- ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341)
+
+## Trivia
+
+- ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) membuat tim kapten pada bulan September 2017 bersamaan dengan sistem [Captain's Pick](/wiki/Beatmaps/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) yang selanjutnya diubah namanya menjadi Project Loved. Hingga akhir 2019, dia mengawasi dan mengelola proyek secara keseluruhan.
+
```
Comment (ReiFan49) [inline wiki/People/The_Team/Project_Loved_Team/id.md:62]: delete the extra space inbetween bullet points.
```suggestion
```

Comment (hikikuman) [reply wiki/People/The_Team/Project_Loved_Team/id.md:62]: okeeii, thank you!

File: wiki/People/The_Team/Project_Loved_Team/id.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Anggota tim lainnya membantu mengontrol kualitas, pemeliharaan, dan pengorganisasian.
+
+## Anggota Tim
+
+<!-- TODO: should probably document old team members, since they aren't credited anywhere, besides Toy in Trivia -->
+
+![][flag_US] **[clayton](https://osu.ppy.sh/users/3666350)** mengawasi proyek, dan menulis berita dan postingan forum, ![][flag_LT] [huu](https://osu.ppy.sh/users/6044237) mengelola spreadsheet publik dan membantu memoderasi diskusi tentang Loved, dan ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), ![][flag_GB] [hypercyte](https://osu.ppy.sh/users/9155377), and ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410) memeriksa metadata dari beatmap yang akan di pilih melalui pemungutan suara. Anggota tim lainnya adalah kapten:
```
Comment (ReiFan49) [inline wiki/People/The_Team/Project_Loved_Team/id.md:21]:
```suggestion
![][flag_US] **[clayton](https://osu.ppy.sh/users/3666350)** mengawasi proyek, dan menulis berita dan postingan forum, ![][flag_LT] [huu](https://osu.ppy.sh/users/6044237) mengelola spreadsheet publik dan membantu memoderasi diskusi tentang Loved, dan ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), ![][flag_GB] [hypercyte](https://osu.ppy.sh/users/9155377), and ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410) memeriksa metadata dari beatmap yang akan dipilih melalui pemungutan suara. Anggota tim lainnya adalah kapten:
```

🤔

File: wiki/People/The_Team/Developers/id.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  - devs
+  - osu!dev
+  - osu!devs
+  - osu! dev
+  - osu! devs
+---
+
+# Developer
+
+Developer osu! menjaga game dan situs web tetap berjalan dengan memperbaiki bug dan menambah fitur baru. Kebanyakan dari mereka adalah karyawan dari ppy Pty Ltd, perusahaan pemilik osu!. Mereka membentuk sebagian besar [osu!team](/wiki/People/The_Team).
```
Comment (troke12) [inline wiki/People/The_Team/Developers/id.md:12]:
```suggestion
Developer osu! menjaga game dan situs web tetap berjalan dengan memperbaiki bug dan menambah beberapa fitur baru. Kebanyakan dari mereka adalah karyawan dari perusahaan ppy Pty Ltd, pengembang dan publisher gim osu!. Mereka juga membentuk sebuah tim yang diberi nama dengan [osu!team](/wiki/People/The_Team).
```

File: wiki/People/The_Team/Developers/id.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+Developer osu! dapat dikenali dari warna penggunanya yaitu pink dan gelar "osu! dev" atau "osu! team".
```
Comment (troke12) [inline wiki/People/The_Team/Developers/id.md:14]:
```suggestion
Developer osu! dapat dikenali dari warna penggunanya yaitu pink dan gelar "osu!dev" atau "osu!team".
```
jangan dispasi

File: wiki/People/The_Team/Developers/id.md (hunk 5 above)
Comment (hikikuman) [reply wiki/People/The_Team/Developers/id.md:12]: oke

File: wiki/People/The_Team/Developers/id.md (hunk 6 above)
Comment (hikikuman) [reply wiki/People/The_Team/Developers/id.md:14]: oke, nanti ku coba recheck yang lain juga biar gak miss

**Review by troke12 — DISMISSED** [review]
ok

**Comment by cl8n** [comment]
also some advice for using git in the future: don't make commits to `master` or use it for PRs, make a new branch first named after the topic (something like `id-update-the-team` for this one). and when writing commit messages, use the first line to explain what/where you edited, so people can tell what's going on (`Update id.md` or the article title by itself doesn't tell me much)

**Comment by hikikuman** [comment]
> * there's a bit of formatting that doesn't match English (ex. first sentence of Beatmap Nominators), seems like most of the articles have something different so I recommend you read through these again and compare to English as you go
> * some of the articles are still going to be outdated if you don't update with changes from English (#4276, #4285, #4281, #4286, #4296, #4316)
> * you can add more tags in indonesian if it would help ppl search for the pages

all of id.md are done compared and changed as referenced from en.md. there's no new addition for the tags.

File: wiki/People/The_Team/Global_Moderation_Team/id.md [hunk 7]
```diff
@@ -30,75 +55,113 @@ Tim Moderasi Global bertanggung jawab atas kesejahteraan chatting / forum dan me
...
-[flag_NL]: /wiki/shared/flag/NL.gif
-[flag_PH]: /wiki/shared/flag/PH.gif
-[flag_PL]: /wiki/shared/flag/PL.gif
-[flag_PT]: /wiki/shared/flag/PT.gif
-[flag_RU]: /wiki/shared/flag/RU.gif
-[flag_SE]: /wiki/shared/flag/SE.gif
-[flag_TR]: /wiki/shared/flag/TR.gif
-[flag_TW]: /wiki/shared/flag/TW.gif
-[flag_US]: /wiki/shared/flag/US.gif
+### Grouped by languages moderated
```
Comment (cl8n) [inline wiki/People/The_Team/Global_Moderation_Team/id.md:102]: no translation for this?

File: wiki/People/The_Team/Beatmap_Nominators/id.md [hunk 8]
```diff
@@ -1,196 +1,241 @@
...
+tags:
+  - BN
+  - BNG
 ---
 
-# Beatmap Nominator
+# Beatmap Nominators
 
-*Lihat juga: [Peraturan Beatmap Nominator](/wiki/Beatmap_Nominator_Rules) dan [Informasi Umum untuk Beatmap Nominator](/wiki/Beatmap_Nominator_General_Information).*
+*Lihat juga: [Peraturan Beatmap Nominator](Rules), [Informasi Umum Tentang Beatmap Nominators](General_Information), dan [Menjadi seorang Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
```
Comment (cl8n) [inline wiki/People/The_Team/Beatmap_Nominators/id.md:9]: missing expectations link

**Review by cl8n — CHANGES_REQUESTED** [review]

File: wiki/People/The_Team/Beatmap_Nominators/id.md (hunk 8 above)
Comment (hikikuman) [reply wiki/People/The_Team/Beatmap_Nominators/id.md:9]: okii

File: wiki/People/The_Team/Global_Moderation_Team/id.md (hunk 7 above)
Comment (hikikuman) [reply wiki/People/The_Team/Global_Moderation_Team/id.md:102]: uhh, sorry my bad. changed to "Dikelompokan berdasarkan bahasa yang dimoderasi"

File: wiki/People/The_Team/Project_Loved_Team/id.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+- ![][flag_US] **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**
+- ![][flag_US] [-mint-](https://osu.ppy.sh/users/8976576)
+- ![][flag_SG] [Abraxos](https://osu.ppy.sh/users/5025064)
+- ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573)
+- ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341)
+
+## Trivia
+
+- ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) membuat tim kapten pada bulan September 2017 bersamaan dengan sistem [Captain's Pick](/wiki/Beatmaps/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) yang selanjutnya diubah namanya menjadi Project Loved. Hingga akhir 2019, dia mengawasi dan mengelola proyek secara keseluruhan.
+- Terdapat grup pengguna Project Loved di situs web dengan ID grup 31, tetapi daftar tersebut tidak bersifat publik. Hal ini digunakan sebagai izin untuk memoderasi [forum Project Loved](https://osu.ppy.sh/community/forums/120) dan mempromosikan beatmap ke kategori Loved, dan anggota tersebut adalah ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) dan ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689).
```
[the remark this answers is not in the document]
Comment (hikikuman) [reply wiki/People/The_Team/Project_Loved_Team/id.md]: added huu on the second point of trivia

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
thnx 👍

**Pull request merged**